qpsolvers_benchmark/benchmark.py

```diff
@@ -46,11 +48,9 @@ def parse_command_line_arguments():
     parser = argparse.ArgumentParser(
         description="Benchmark quadratic programming solvers"
     )
-    parser.add_argument(
-        "test_set",
-        choices=TEST_SETS,
-        help="test set from the benchmark to consider",
-    )
+    parser.add_argument("test_set_path",
```

Comment:
LGTM but the indentation will not pass the [Black](https://github.com/psf/black) check in the CI.